From c142dcdbfe1e72211e79b9de797f8b91a40eb330 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 7 May 2019 19:50:25 +0200
Subject: [PATCH 2/2] [DNM] Squash of lineage-16.0-android-9.0.0_r37

Squashed commit of the following:

commit 33a187438ef781129865711235b2ab4d6a7ec69d
Merge: b0a428fd0 943d60667
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Feb 27 04:15:50 2019 +0000

    Snap for 5339334 from 943d60667ab79c81ab65446b04a2c48145944115 to pi-qpr3-release

    Change-Id: I9491d1637d11975fe1f13ca6bdd8dd58dfca1295

commit 943d60667ab79c81ab65446b04a2c48145944115
Author: Bill Yi <byi@google.com>
Date:   Thu Feb 21 22:07:35 2019 -0800

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I4545cc06e123a46f1b3ef6c659d7754d2c5cfd52

Change-Id: I8270cdb56e3b5be21e43bea1c0e3130d5cea3444
---
 apps/Fallback/res/values-en-rXC/strings.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/apps/Fallback/res/values-en-rXC/strings.xml b/apps/Fallback/res/values-en-rXC/strings.xml
index 8e3e8aeb2..121fe541a 100644
--- a/apps/Fallback/res/values-en-rXC/strings.xml
+++ b/apps/Fallback/res/values-en-rXC/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="appTitle" msgid="161410001913116606">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‎‏‎‏‏‏‏‏‏‎‏‎‏‎‎‎‏‏‏‏‎‏‎‏‏‏‎‎‎‏‏‎‎‎‏‎‏‎‏‎‏‎‎‎‏‏‎‎‏‏‏‎‎‏‏‏‏‏‎‏‏‏‏‎‏‏‏‏‏‎‎Fallback‎‏‎‎‏‎"</string>
-    <string name="title" msgid="8156274565006125136">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‎‎‏‏‎‎‎‎‏‏‏‎‏‎‎‎‏‎‎‎‎‎‎‎‏‏‎‎‏‎‎‏‏‎‏‏‏‏‏‎‎‏‏‎‏‎‎‎‎‏‎‏‎‎‎‎‎Unsupported action‎‏‎‎‏‎"</string>
-    <string name="error" msgid="6539615832923362301">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‏‏‎‏‎‏‏‎‎‎‎‎‏‎‏‏‎‎‎‎‏‏‏‏‏‎‎‏‏‏‎‏‎‎‎‎‎‏‏‏‎‏‎‏‎‏‏‎‏‎‎‏‏‏‏‏‏‏‏‎‏‎That action is not currently supported.‎‏‎‎‏‎"</string>
+    <string name="appTitle" msgid="161410001913116606">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‎‏‎‏‏‏‏‏‏‎‏‎‏‎‎‎‏‏‏‏‎‏‎‏‏‏‎‎‎‏‏‎‎‎‏‎‏‎‏‎‏‎‎‎‏‏‎‎‏‏‏‎‎‏‏‏‏‏‎‏‏‏‏‎‏‏‏‏‏‎‎Fallback‎‏‎‎‏‎"</string>
+    <string name="title" msgid="8156274565006125136">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‎‎‏‏‎‎‎‎‏‏‏‎‏‎‎‎‏‎‎‎‎‎‎‎‏‏‎‎‏‎‎‏‏‎‏‏‏‏‏‎‎‏‏‎‏‎‎‎‎‏‎‏‎‎‎‎‎Unsupported action‎‏‎‎‏‎"</string>
+    <string name="error" msgid="6539615832923362301">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‏‏‎‏‎‏‏‎‎‎‎‎‏‎‏‏‎‎‎‎‏‏‏‏‏‎‎‏‏‏‎‏‎‎‎‎‎‏‏‏‎‏‎‏‎‏‏‎‏‎‎‏‏‏‏‏‏‏‏‎‏‎That action is not currently supported.‎‏‎‎‏‎"</string>
 </resources>
-- 
2.17.1

